app/models/rule.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class Rule < ApplicationRecord
+  audited except: %i[created_at updated_at locked], max_audits: 1000
```

Comment:
why would we not audit who is locking/unlocking controls? Is the idea to only audit the core fields of the rules (title, desc, fix/check text, justification)?

Reply:
Yes the point of the audit functionality is to revert, not really to "audit" and reverting a change that is a switch seems silly